Makefile

```diff
@@ -62,7 +65,13 @@ ifdef SHOULD_LINT
 	@[ ! -s "$(FMT_LOG)" ] || (echo "gofmt failed:" | cat - $(FMT_LOG) && false)
 
 	$(eval VET_LOG := $(shell mktemp -t govet.XXXXX))
-	@go vet $(PACKAGES) 2>&1 | grep -v '^exit status' | $(FILTER_LINT) > $(VET_LOG) || true
+	@$(foreach pkg, $(PACKAGES), \
+		go tool vet $(VET_RULES) $(pkg) 2>&1 | \
+			grep -v '^exit status' | \
+			$(FILTER_LINT) | \
+			grep -v 'something"` not compatible with reflect.StructTag.Get:' \
```

Comment:
Is this last one from your other PR? can you defer adding this until that PR and we can evaluate whether there's a better way to do this.